bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/config/CodeCheckerContext.java

```diff
@@ -343,12 +348,13 @@ public void runReportJob(ReportListView target, String currentFileName) {
             return;
         }
 
-        Logger.log(IStatus.INFO, "Started Filtering Reports for project: "+project.getName());
+        Logger.log(IStatus.INFO, "Started Filtering Reports for project: " + project.getName() + " " + currentFileName);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String " " appears 2 times in the file.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641620)